typo: Update quickstart-create-mysql-server-database-using-azure-cli.md

zeneli opened the pull request:
In section `Configure SSL settings` there is a typo in the Azure CLI prompt.
It is: `az mysql server update --resource-group myresourcegroup --name myserver4demo -g -n --ssl-enforcement Disabled`
It should be: `az mysql server update --resource-group myresourcegroup --name myserver4demo --ssl-enforcement Disabled`
(remove the redundant `-n` and `-g` CLI parameters)

azurecla commented:
Hi __@jimsparkman__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

zeneli committed:
Update quickstart-create-mysql-server-database-using-azure-cli.md

In section `Configure SSL settings` there is a typo in the Azure CLI prompt.
It is: `az mysql server update --resource-group myresourcegroup --name myserver4demo -g -n --ssl-enforcement Disabled`
It should be: `az mysql server update --resource-group myresourcegroup --name myserver4demo --ssl-enforcement Disabled`
(remove the redundant `-n` and `-g` CLI parameters)
